packages/qwik/src/actions/formAction$.ts

```diff
@@ -83,8 +83,8 @@ export function formActionQrl<
         // Get form values from form or JSON data
         const values: PartialValues<TFieldValues> =
           type === 'application/x-www-form-urlencoded' ||
-          type === 'multipart/form-data'
-            ? getFormDataValues(await event.request.formData(), formDataInfo)
+            type === 'multipart/form-data'
+            ? getFormDataValues(event.sharedMap.get('@actionFormData'), formDataInfo)
```

Comment:
Is this somewhere properly documented in the Qwik docs? I this the right way to access `FormData`? I don't want to add a "workaround" to the Modular Forms code.

Reply:
Hello, I found that here:
https://github.com/QwikDev/qwik/issues/3190